pkg/reconciler/instances/istio/actions/performer.go

```diff
@@ -197,6 +198,11 @@ func (c *DefaultIstioPerformer) Install(context context.Context, kubeConfig, ist
 
 	err = commander.Install(mergedCNI, kubeConfig, logger)
 	if err != nil {
+		// In case of error in the istioctl, old mutatingwebhook won't be deactivated, which will block later reconciliations.
+		err2 := webhooks.DeleteConflictedDefaultTag(context, c.provider, kubeConfig, logger)
+		if err2 != nil {
```

Comment:
If we don't want to wrap err2 into err I think we should at least log err2 on error level.